core/controllers/base.py

```diff
@@ -295,8 +299,81 @@ def dispatch(self):
                 self.handle_exception(e, self.app.debug)
                 return
 
+        schema_validation_succeeded = True
+        try:
+            self.validate_args_schema()
+        except self.InvalidInputException as e:
+            self.handle_exception(e, self.app.debug)
+            schema_validation_succeeded = False
+        # TODO(#13155): Remove this clause once all the handlers have had
+        # schema validation implemented.
+        except NotImplementedError as e:
+            self.handle_exception(e, self.app.debug)
+            schema_validation_succeeded = False
+
+        if schema_validation_succeeded is False:
+            return
```

Comment:
The reason behind I have used an extra variable is that in the dispatch function there are already 5 return statements, if I add 2 more then it will cross max return statements i.e, 6.
The error I faced is,
“R:244, 4: Too many return statements (7/6) “